src/services/leafletHelpers.js

```diff
@@ -658,6 +658,11 @@ angular.module('ui-leaflet').service('leafletHelpers', function ($q, $log) {
                 }
             }
```

Comment:
Like this:
https://github.com/angular-ui/ui-leaflet-draw/blob/master/src/drawDirective.coffee#L2-L15

But yours will look like: 

```js
angular.module('ui-leaflet').config(function($provide) {
  $provide.decorator('leafletHelpersProvider', function($delegate) {});
  $delegate.MapboxGL = {
    isLoaded: function() {
      return angular.isDefined(L.mapboxGL);
    }
  };
  return $delegate;
});

```